dashboard_documentation.md

```diff
@@ -69,6 +69,78 @@ Create New Site
    <img class="my-3" src="/images/plant_site.png"/>
 
 
+Create New Aggregate
+--------------------
+{: .anchor}
+
+1. Navigate to the aggregates listing page using the **Aggregates** link in
+   the left sidebar. At the top of the listing, click **Create new Aggregate**.
+
+   <img class="my-3" src="/images/aggregate_listing.png"/>
+
+2. Enter the metadata for your Aggregate.
+
+   <img class="my-3" src="/images/aggregate_form.png"/>
+
+3. After submission, you will be redirected to an Aggregate page which displays
+   the observations that make up the aggregate (see 
+   [Add Observations to an Aggregate](#add-observations-to-an-aggregate)).
+
+   <img class="my-3" src="/images/aggregate_page.png"/>
+
+
+### Add Observations to an Aggregate
+{: .anchor}
+
+Each aggregate is defined by the observations it contains. Before you can add
```

Comment:
Observations must be defined before they can be added to an aggregate (see Create New Observation)